webui/react/src/pages/ResourcePool/Topology.tsx

```diff
@@ -59,6 +59,19 @@ const NodeElement: React.FC<PropsWithChildren<NodeElementProps>> = ({ name, slot
 };
 
 const Topology: React.FC<PropsWithChildren<Props>> = ({ nodes }) => {
+  const MAX_USABLE_NODES = 100;
+  if (nodes.length > MAX_USABLE_NODES) {
+    return (
+      <Section title="Topology">
+        <div className={css.warningContainer}>
+          <p>
+            Current Topology view is disabled for pools with more than {MAX_USABLE_NODES} nodes.
```

Comment:
I think the updated suggestion was that we just hide this @hkumar92

Reply:
yep should be fixed